docs/developer-guide/standardize-python-project-configuration.md

```diff
@@ -0,0 +1,103 @@
+---
+myst:
+  html_meta:
+    "description": "Standardize project configuration in Plone with plone/meta"
+    "property=og:description": "Standardize project configuration in Plone with plone/meta"
+    "property=og:title": "Standardize project configuration in Plone with plone/meta"
+    "keywords": "Plone 6, standardize, project, configuration, development, plone/meta"
+---
+
+# Standardize Python project configuration
+
+This part of the documentation describes how to standardize Python project configuration in Plone.
+It does not cover the following.
+
+-   Volto or any other JavaScript-based project, which has its own ecosystem.
+-   Monorepos, such as [Cookieplone](https://github.com/plone/cookieplone).
```

Comment:
I would remove this line, as `Cookieplone` is not actually a monorepo.

Reply:
Or rewrite it to something like:
Monorepo projects with backend and frontend codebases, like the ones created by cookieplone

Reply:
Done in a27eaa2c5c0abff8f0d7bf3f5ed15ef9093fd43f